app/src/main/java/com/example/henzoshimada/feeltrip/Mood.java

```diff
@@ -33,6 +35,9 @@
     private static boolean delState; // 1 for delete, 0 for add
     //TODO: Test deleting a mood from ES after popping from local queue
 
+    private int emoji;
+    private String append;
+
```

Comment:
New fields to hold on elasticsearch. Allows us to do much more with the UI display.